### PR TITLE
Unify classes

### PR DESCRIPTION
**Pull request by GorogPeter**
I unified the Load32/Load64 classes into SimpleLoad, and Store32/Store64 classes into SimpleStore.

### CODE REVIEW

File: src/interpreter/ByteCode.h [hunk 1]
```diff
@@ -399,52 +400,25 @@ class Load32 : public ByteCode {
         DUMP_BYTECODE_OFFSET(dstOffset);
     }
 
-    virtual size_t byteCodeSize()
+    virtual void byteCodeSize()
```
Comment (zherczeg) [inline src/interpreter/ByteCode.h:403]: Return value should not be changed to `void`

**Review by zherczeg — CHANGES_REQUESTED** [review]
Nice patch

File: src/interpreter/ByteCode.h (hunk 1 above)
Comment (GorogPeter) [reply src/interpreter/ByteCode.h:403]: Thank you!
I'm working on it.

**Comment by GorogPeter** [comment]
I've done the changes.

**Review by clover2123 — APPROVED** [review]
LGTM 👍

**Pull request merged**